packages/common/container-definitions/src/runtime.ts

```diff
@@ -197,6 +197,17 @@ export interface IContainerContext {
 	updateDirtyContainerState(dirty: boolean): void;
 
 	readonly supportedFeatures?: ReadonlyMap<string, unknown>;
+
+	/**
+	 * WARNING: this id is meant for telemetry usages ONLY, not recommended for other consumption
+	 * This id is not supposed to be exposed anywhere else. It is dependant on usage or drivers
+	 * and scenarios which can change in the future.
+	 * @deprecated - 2.0.0-internal.5.2.0 - The docId is already logged by the IContainerContext.taggedLogger for
```

Comment:
```suggestion
	 * @deprecated 2.0.0-internal.5.2.0 - The docId is already logged by the IContainerContext.taggedLogger for
```

`@deprecated` blocks don't want a `-` separator. See: https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#deprecated

I would also recommend using an `{@link}` tag where we reference `IContainerContext.taggedLogger` for the generated docs.